RELEASE_PROCESS.md

```diff
@@ -101,19 +88,6 @@ The git SHA of the build you are testing can be found in
 
 ### Preparing for the release
 
-- [ ] Check that omnibus_overrides.rb has the correct version given
-  the type of changes in this release.
-
-- [ ] Download the previous release of Chef Server (.deb).
-
-- [ ] Run `./dev/scripts/update-changelog.sh /path/to/deb/from/previous/step`
-
-- [ ] Grab the section on the latest version from `NEW_CHANGELOG.md` and paste
-  it at the top of `CHANGELOG.md`.
-
-- [ ] Copy the components section from `MODIFIED_COMPONENTS_CHANGELOG.md` and
```

Comment:
Yup!  We use `omnibus`'s old changelog generator to create the "Components" and "Contributors" section of the changelong:

https://github.com/chef/chef-server/blob/master/CHANGELOG.md#components-3